src/cdomains/apron/apronDomain.apron.ml

```diff
@@ -295,27 +295,27 @@ struct
     (* partition assigns with supported and unsupported exps *)
     let (supported, unsupported) =
       ves
-      |> List.enum
-      |> Enum.map (Tuple2.map2 (fun e ->
+      |> List.to_seq
+      |> Seq.map (Tuple2.map2 (fun e ->
           match Convert.texpr1_of_cil_exp ask nd env e (Lazy.from_val no_ov) with
           | texpr1 -> Some texpr1
           | exception Convert.Unsupported_CilExp _ -> None
         ))
-      |> Enum.partition (fun (_, e_opt) -> Option.is_some e_opt)
+      |> Seq.partition (fun (_, e_opt) -> Option.is_some e_opt)
```

Comment:
There maybe should be `Seq.memoize` before/after this stuff to avoid duplicating computations when working on `supported` and `unsupported` later on. The documentation for `Seq.partition` warns about this.